layouts/partials/template-page.html

```diff
@@ -82,7 +82,7 @@ <h2>
        {{ $repo_name := $repo_json.full_name}}
        {{ $repo_date := $repo_json.updated_at}}
        <div class="github">
-        <a href="{{$repo_url}}">
+        <a href="{{.repository_url}}">
```

Comment:
just to make sure i understand, the `$repo_url` is a substitution on `repository_url` that is set to `api.github.com/repos`, right? makes sense to use `repository_url` for a link instead, just want to make sure i get what's going on

Reply:
correct

`repo_url` is modified off of `repository_url` which comes from the registry. I should improve variable naming

Reply:
All fixed :)